src/views/partials/footer.ejs

```diff
@@ -1,7 +1,25 @@
-</main>
-<!--Todo: Footer-->
+<footer>
 <% if (typeof libs !== 'undefined') { %>
```

Comment:
Script tags need to be right above body, so just cut paste this right before the closing body tag